sites/diverseeducation.com/server/templates/static-pages/mppwsa-yearly.marko

```diff
@@ -19,8 +19,13 @@ $ const profiles = currentYearData ? currentYearData.profiles: title;
 $ const promisingPractices = currentYearData ? currentYearData.promisingPractices: title;
 $ const advisoryBoard = currentYearData ? currentYearData.advisoryBoard: title;
 $ const promisingPlaces = currentYearData ? currentYearData.promisingPlaces: title;
+$ const commitment = currentYearData ? currentYearData.commitment: title;
+$ const encourageWork = currentYearData ? currentYearData.encourageWork: title;
+$ const fostering = currentYearData ? currentYearData.fostering: title;
 $ const chee = currentYearData ? currentYearData.chee: title;
 $ const ideas = currentYearData ? currentYearData.ideas: title;
+$ const nationalPerspective = currentYearData ? currentYearData.ideas: title;
```

Comment:
This appears to be looking for the wrong property here and should likely be changed to `const nationalPerspective = currentYearData ? currentYearData.nationalPerspective : title;`